packages/cardano-services/src/ChainHistory/DbSyncChainHistory/mappers.ts

```diff
@@ -105,14 +105,30 @@ export const mapWithdrawal = (withdrawalModel: WithdrawalModel): Cardano.Withdra
 // Remove this and select the actual redeemer data from `redeemer_data` table.
 const stubRedeemerData = Buffer.from('not implemented');
 
+const mapRedeemerPurpose = (purpose: string): Cardano.RedeemerPurpose => {
+  switch (purpose) {
+    case 'cert':
+      return Cardano.RedeemerPurpose.certificate;
+    case 'mint':
+      return Cardano.RedeemerPurpose.mint;
+    case 'spend':
+      return Cardano.RedeemerPurpose.spend;
+    case 'reward':
+      return Cardano.RedeemerPurpose.withdrawal;
+    default:
```

Comment:
Please also include the 2 conway specific purpose types: 'voting' and 'proposing': https://github.com/IntersectMBO/cardano-db-sync/blob/sancho-4.1.0/doc/schema.md

Reply:
Added new purpose types to core redeemer type and updated db-sync mapper. Please re-review